bin/cordova_plist_to_config_xml

```diff
@@ -101,7 +106,7 @@ def UpdateProjectFile(path):
     if 'Cordova.plist' in line:
       line = line.replace('Cordova.plist', 'config.xml')
       line = line.replace('lastKnownFileType = text.plist.xml', 'lastKnownFileType = text.xml')
-    print line,
+    print(line, end='')
```

Comment:
Why is it important to overwrite the newline character at the end of print here? Does this need to print in one line? Else `end=''` could be left out

Reply:
Did you forget this line?  `from __future__ import print_function`

Reply:
Yeah, you're right. Never worked with that before